napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -327,7 +327,12 @@ def test_points_layer_display_correct_slice_on_scale(make_napari_viewer):
     pts.add((8.7, 0, 0))
     viewer.dims.set_point(0, 30 * 0.29)  # middle plane
     layer = viewer.layers[1]
-    indices, scale = layer._slice_data(layer._slice_indices)
+
+    request = pts._make_slice_request_internal(
+        layer._slice_input, layer._slice_indices
+    )
+    response = request()
+    indices = response.indices
     np.testing.assert_equal(indices, [0])
```

Comment:
This is a bit more concise and likely future-proof too.

```suggestion
    request = pts._make_slice_request(viewer.dims)
    response = request()
    np.testing.assert_equal(response.indices, [0])
```